packages/zoe/docs/zoe-zcf.md

```diff
@@ -0,0 +1,57 @@
+## Zoe and ZCF
+
+At the current time, Zoe is split into an "upper" part, which is the facet that the contract communicates with, and the "middle", which contains all the common pieces shared across all contracts. The middle has the InviteIssuer, manages the purses for all contract instances, and is the central point of contact for installing and instantiating contracts. Each time a contract is instantiated with makeInstance(), a new vat is created, and the code for ZCF (the Zoe Contract Facet) is executed, immediately running the code for the contract itself. 
```

Comment:
Let's not use upper and middle

Reply:
okay. I didn't find consensus terminology for zoe main, so I made something up.